glad/lang/c/loader/__init__.py

```diff
@@ -3,9 +3,6 @@
 %(pre)s void* %(proc)s(const char *namez);
 
 #if defined(_WIN32) || defined(__CYGWIN__)
-#ifndef _WINDOWS_
```

Comment:
I think we still need this block, in case `APIENTRY` is already defined from somewhere that is not `windows.h`.